### PR TITLE
Add aggregation for CDC restricted data

### PR DESCRIPTION
**Pull request by vanshkumar**
Add aggregation logic for joining CDC restricted data with the public state_fips dataset & the ACS population tables. Also fix a standardization issue in the cdc_restricted local script to make county_fips a string.

Note this also reverts the "_staging" table naming change as dedupe is not yet complete.

### CODE REVIEW

File: config/data_sources/AGG_cdc_restricted.sql [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+-- County-level joins with ACS population tables. We first do the same join as
+-- in the state case to get state fips codes and use ACS state & county names,
+-- but also join with the ACS population table to get ACS county names. We then
+-- join again with ACS, this time to get county x {race, sex, age} data.
+CREATE OR REPLACE TABLE cdc_restricted_data.by_race_county AS
+WITH cdc_restricted_race_county AS (
+    SELECT DISTINCT
+        a.county_fips,
+        IF(a.county_fips = "", "Unknown", c.county_name) as county_name,
+        IF(a.state_name = "Unknown", "", b.state_fips_code) as state_fips,
```
Comment (ldbnr4) [inline config/data_sources/AGG_cdc_restricted.sql:84]: a.state_fips? same below

File: python/datasources/cdc_restricted.py [hunk 2]
```diff
@@ -27,21 +27,25 @@ def write_to_bq(self, dataset, gcs_bucket, **attrs):
             raise ValueError('filename passed to write_to_bq is not a '
                              'comma-separated list of files')
         files = gcs_files.split(',')
+        print("Files that will be written to BQ:", files)
```
Comment (ldbnr4) [inline python/datasources/cdc_restricted.py:30]: was this for dev?

**Review by ldbnr4 — CHANGES_REQUESTED** [review]

**Review by aaronsn — APPROVED** [review]

File: config/data_sources/AGG_cdc_restricted.sql [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+FROM cdc_restricted_age_state AS x
+LEFT JOIN `acs_population.by_age_state` AS y ON
+  x.state_fips = y.state_fips AND
+  x.age = y.age
+;
+
+
+-- County-level joins with ACS population tables. We first do the same join as
+-- in the state case to get state fips codes and use ACS state & county names,
+-- but also join with the ACS population table to get ACS county names. We then
```
[the remark this answers is not in the document]
Comment (vanshkumar) [reply config/data_sources/AGG_cdc_restricted.sql]: good point, done (note that fips_codes_all doesn't have two-letter state abbreviations so we still need 2 joins)

File: config/data_sources/AGG_cdc_restricted.sql [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+SELECT
+    x.state_fips, x.state_name,
+    x.race_and_ethnicity,
+    x.cases, x.hosp_y, x.hosp_n, x.hosp_unknown, x.death_y, x.death_n, x.death_unknown,
+    y.population
+FROM cdc_restricted_race_state AS x
+LEFT JOIN `acs_population.by_race_state_std` AS y ON
+  x.state_fips = y.state_fips AND
+  x.race_and_ethnicity = y.race_and_ethnicity
+;
```
[the remark this answers is not in the document]
Comment (vanshkumar) [reply config/data_sources/AGG_cdc_restricted.sql:26]: done

File: config/data_sources/AGG_cdc_restricted.sql [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+CREATE OR REPLACE TABLE cdc_restricted_data.by_race_state AS
+WITH cdc_restricted_race_state AS (
+    SELECT DISTINCT
+        b.state_fips_code as state_fips,
+        IF(a.state_name = "Unknown", "Unknown", b.state_name) as state_name,
+        a.race_and_ethnicity,
+        a.cases, a.hosp_y, a.hosp_n, a.hosp_unknown, a.death_y, a.death_n, a.death_unknown
+    FROM `cdc_restricted_data.cdc_restricted_by_race_state` AS a
+    LEFT JOIN `bigquery-public-data.census_utility.fips_codes_states` AS b
+        ON a.state_name = b.state_postal_abbreviation
```
[the remark this answers is not in the document]
Comment (vanshkumar) [reply config/data_sources/AGG_cdc_restricted.sql]: done

File: python/datasources/cdc_restricted.py (hunk 2 above)
Comment (vanshkumar) [reply python/datasources/cdc_restricted.py:30]: yeah, it was, but honestly i think it's good to keep in as logging. let me know if there's a better way to log things

File: config/data_sources/AGG_cdc_restricted.sql (hunk 1 above)
Comment (vanshkumar) [reply config/data_sources/AGG_cdc_restricted.sql]: i think the line you're referring to is the one that actually gets state_fips for the table; we don't yet have access to it which is why this uses a.state_postal

File: config/data_sources/AGG_cdc_restricted.sql [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+
+-- State-level joins with ACS population tables. First, we join with a public
+-- state fips codes dataset so we can convert from two-leter state postal
+-- abbreivation to state fips and full state name (eg AL -> 01 and "Alabama").
+-- Then we join with ACS on state x {race, sex, age}.
+CREATE OR REPLACE TABLE cdc_restricted_data.by_race_state AS
+WITH cdc_restricted_race_state AS (
+    SELECT DISTINCT
+        b.state_fips_code as state_fips,
+        IF(a.state_name = "Unknown", "Unknown", b.state_name) as state_name,
```
[the remark this answers is not in the document]
Comment (vanshkumar) [reply config/data_sources/AGG_cdc_restricted.sql]: Yeah, generally speaking I have tried to use NULL to represent the kind of mismatch you're describing (eg in the join w/ the ACS population table, if state_fips & race don't show up in the population table, the population value is NULL).

**Review by ldbnr4 — APPROVED** [review]
lgtm

**Pull request merged**